WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtils.java

```diff
@@ -361,14 +361,14 @@ public void onResponse(JSONObject jsonObject) {
                             switch (userError.optString("code")) {
                                 case "invalid_input":
                                     switch (userError.optString("message")) {
-                                        case "User not found":
-                                            callback.onUsernameValidation(username, ValidationResult.USER_NOT_FOUND);
-                                            continue;
                                         case "Invalid email":
                                             callback.onUsernameValidation(username, ValidationResult.INVALID_EMAIL);
                                             continue;
+                                        case "User not found":
```

Comment:
WDYT about adding a small comment here like "fall through to the default case" to make it explicit?